### ISSUE TITLE
use of deprecated item 'failure::Error::causes'

### ISSUE THREAD

**Issue by spease** [killercup/quicli#97]
Looks like quicli needs to be updated for the latest use of failure.

```
warning: use of deprecated item 'failure::Error::causes': please use the 'iter_chain()' method instead                                                                                                                                                                         
   --> src/main.rs:46:1                                                                                                                                                                                                                                                        
    |                                                                                                                                                                                                                                                                          
46  | / main!(|args: Cli, log_level: verbosity| {                                                                                                                                                                                                                              
47  | |     let client = Client::connect(&args.host, args.port)?;                                                                                                                                                                                                              
48  | |     debug!("DATABASES: {:?}", client.database_names());                                                                                                                                                                                                                
49  | |     let db = client.db(&args.db);                                                                                                                                                                                                                                      
...   |                                                                                                                                                                                                                                                                        
121 | |     */                                                                                                                                                                                                                                                                 
122 | | });                                                                                                                                                                                                                                                                    
    | |___^                                                                                                                                                                                                                                                                    
    |                                                                                                                                                                                                                                                                          
    = note: #[warn(deprecated)] on by default                                                                                                                                                                                                                                  
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)                                                                                     ```

**Comment by kimsnj** [comment]
The warning should be fixed in the latest version of quicli: [0.3.1](https://github.com/killercup/quicli/releases/tag/v0.3.1)

**Comment by killercup** [comment]
Yep, should be fixed by 0.3.1.
